src/shared/modules/entry.ts

```diff
@@ -21,3 +21,7 @@ export function normalizeDestination(destination = '') {
 export function isUsersFolder(key = '') {
     return key.toLowerCase() === 'users/';
 }
+
+export const isFavoritesAliasAvailable = (alias?: string | null) => {
+    return alias !== null && alias !== undefined && alias !== '';
```

Comment:
Why is so complex?
```
return Boolean(alias)
```

Reply:
Good catch, alias is a string and '0' will not break logic, fixed